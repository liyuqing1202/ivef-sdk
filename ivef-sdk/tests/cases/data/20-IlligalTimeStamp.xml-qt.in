<MSG_Pong xmlns="urn:http://www.ivef.org/XMLSchema/IVEF/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:http://www.ivef.org/XMLSchema/IVEF/1.1
\\projsrvr\isis\develop\ivef-sdk\trunk\doc\ivef1.xsd">
	<Header MsgRefId="String" Version="String"/>
	<Body>
		<Pong MsgId="String" TimeStamp="2001-12-17T09:30:47.001" SourceId="-0"/>
	</Body>
</MSG_Pong>

