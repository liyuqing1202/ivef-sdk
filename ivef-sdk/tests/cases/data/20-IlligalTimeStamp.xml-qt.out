TestApp ready for input
MSG_Pong
    Header
        Version = String
        MsgRefId = String
    Body
        Pong
            TimeStamp = 2001-12-17T09:30:47.001Z
            MsgId = String
            SourceId = 0
<MSG_Pong>
<Header Version="String" MsgRefId="String"/>
<Body>
<Pong TimeStamp="2001-12-17T09:30:47.001Z" MsgId="String" SourceId="0"/>
</Body>
</MSG_Pong>

TestApp shutting down
